flight/Libraries/inc/morsel.h

```diff
@@ -0,0 +1,33 @@
+// Morse Library header
```

Comment:
It's imported from OpenLager, which I wrote using BSD-ish conventions.   Can reformat if you want (in which case I should reformat over there, so things stay in sync).